config/application.rb

```diff
@@ -1,16 +1,10 @@
 require File.expand_path('../boot', __FILE__)
 
-# Pick the frameworks you want:
-require "rails/all"
-require 'tilt'
-# require "rails/test_unit/railtie"
+require 'rails/all'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.